### DISCUSSION TITLE
Tensorboard not working?

### DISCUSSION THREAD

**Discussion by KhyamAllami** [acids-ircam/RAVE#121] [Q&A] [answered]
Hi all, is anyone else having trouble with the Tensorboard in the Colab?
I just have a 403 error page. It doesn't seem to be working at all.
Thanks,
Khyam

**Comment by wwerkk** [comment, marked as answer]
Hi!
I notice the same issue whenever I have one of my multiple cookie/tracking/ad blockers on, worth giving a shot to whitelist Colab

**Reply by KhyamAllami** [reply]
That did the trick. Thanks a lot.
